CyberHealth/templates/scripts/govuk/_bottom.html

```diff
@@ -2,4 +2,8 @@
     <script src="{% static 'scripts/all.js'%}"></script>
     <script>
         window.GOVUKFrontend.initAll()
+<<<<<<< HEAD
     </script>
+=======
+    </script>
+>>>>>>> develop
```

Comment:
Something funny github related going on here - the suggestion editor is acting up because of the git tags but this should just be a single `</script>` tag!

Reply:
That's a stray conflict resolution

Can we please go again